cli/tsc/mod.rs

```diff
@@ -829,7 +829,7 @@ pub fn exec(request: Request) -> Result<Response, AnyError> {
   let mut runtime = JsRuntime::new(RuntimeOptions {
     startup_snapshot: Some(compiler_snapshot()),
     extensions: vec![Extension::builder("deno_cli_tsc")
-      .ops(get_tsc_ops())
+      .ops(deno_ops())
```

Comment:
Okay I don't understand this bit... Does `deno_core::ops!()` macro expand to `deno_ops` function?

Reply:
Yep. In this case when I fold the ops into the extension, that will disappear.